test/data.jl

```diff
@@ -262,13 +262,13 @@ module TestData
 
     # # TODO: Restore this functionality
     # m1 = join(dt1, dt2, on = [:a,:b])
-    # @test isequal(m1[:a], Vector(["x", "x", "y", "y", fill("x", 5)]))
+    # @test m1[:a] == Vector(["x", "x", "y", "y", fill("x", 5)]))
```

Comment:
John commented this out 4 years ago https://github.com/JuliaData/DataTables.jl/commit/c5a3d287de2f3fc55101e1c94495625b27a4eb64